Default explanation text for `mobile-center test` is poorly formatted

bryukov opened the pull request:
Fix for #93 

1. Fixed the explanation text formatting by setting the `colWidths` on the `noTableBorders` settings set. `colWidths` are calculated from the current console width.

2. Fixed another problem which also broke output formatting in some consoles (e.g. cmder and powershell). It was caused by the presence of the CRLF characters in some of the category.txt files. cli-table2 doesn't take CR into account (i.e. doesn't removes it when wrapping strings to fit into table cell), thus breaking the formatting.

Here is how it looks like after the fix (powershell console as an example):

![powershell](https://cloud.githubusercontent.com/assets/25530829/22602833/f0b61c6e-ea55-11e6-8685-98bb292f7791.PNG)

Please, note that there is no space between rightmost top cell and the one below it (see the red line on the next image):

![powershell_redline](https://cloud.githubusercontent.com/assets/25530829/22603303/e2119f06-ea57-11e6-9312-9c0b8c5ce1ae.png)

Is it by design? Or should I add a horizontal space between them?

christav commented:
Please add a space between the two categories - it's too hard to read without it.

christav commented on src/util/interaction/out.ts:
```diff
@@ -116,14 +118,26 @@ export function table(...args: any[]): void {
+  let colWidth = Math.floor(availableWidth / tableColumnsCount);
+  let remainderWidth = availableWidth % tableColumnsCount;
+  // Spreading remainder width across the first columns
+  let colWidths = Array<number>(remainderWidth).fill(colWidth + 1).concat(Array<number>(tableColumnsCount - remainderWidth).fill(colWidth));
```
I don't think we want to evenly divide the space - the left column in this case (the category name) is probably narrower than the text. Perhaps pass in the desired width for the first column and let the rest of the space go to the right column?

christav replied:
Also, I'm fine leaving this function hard-wired for the help system at the moment.

bryukov commented:
@christav I have addressed your comments in another commit. Can you, please, take a look on it?

The help messages for "test" and "apps set-current" after the changes look like this:
![iteration2](https://cloud.githubusercontent.com/assets/25530829/22657335/2d25aeb4-eca7-11e6-8171-e87471d9d177.PNG)
